modules/transfer-server/main.tf

```diff
@@ -80,8 +80,8 @@ check "endpoint_details_configuration" {
 
 check "vpc_endpoint_requirements" {
   assert {
-    condition     = var.endpoint_type != "VPC" || (var.endpoint_details != null && var.endpoint_details.vpc_id != null && var.endpoint_details.subnet_ids != null)
-    error_message = "When endpoint_type is 'VPC', endpoint_details with vpc_id and subnet_ids must be provided."
+    condition     = var.endpoint_type != "VPC" || (var.endpoint_details != null)
```

Comment:
@prabirsekhri @SamuelOuvrard is there any option to move just these functional changes in the `modules/transfer-server` forward and to cut a new release. I know you want to polish up the examples but could you move that effort into a separate PR and just move this small set up validation and functional changes in `main.tf` and `variables.tf` and cut a new release if possible.


My customers have been waiting for these changes so we can merge our downstream changes.

Much appreciated!

Reply:
noted, we are trying to streamline this for all users and these will be merged as part of the release